core-web/libs/data-access/src/lib/dot-properties/dot-properties.service.ts

```diff
@@ -57,13 +57,21 @@ export class DotPropertiesService {
 
     /**
      * Get the value of specific feature flag
-     *
+     * Some features we want to display in case of not found.
      * @param {FeaturedFlags} key
      * @return {*}  {Observable<boolean>}
      * @memberof DotPropertiesService
      */
-    getFeatureFlag(key: FeaturedFlags): Observable<boolean> {
-        return this.getKey(key).pipe(map((value) => value === 'true'));
+    getFeatureFlag(key: FeaturedFlags, onNotFound = false): Observable<boolean> {
```

Comment:
Add a test case for then new param

Reply:
`onNotFound` sounds like a callback to me. What about `shouldAlwaysShow`?